docs/sources/integrations/uptimerobot/index.md

```diff
@@ -55,9 +55,9 @@ POST Value (JSON Format):
 
 1. Click Add New Monitor
 1. Monitor Type HTTP(s)
-1. Friendly Name Test Amixr
-1. Set URL to <http://devnull.amixr.io> or any other non-existent domain
-1. Click Checkbox next to Amixr Alert Contact (created in the previous step)
+1. Friendly Name Test OnCall
+1. Set URL to <http://devnull.oncall.io> or any other non-existent domain
+1. Click Checkbox next to OnCall Alert Contact (created in the previous step)
```

Comment:
aside from these changes the rest of the changes in this file are IDE autoformatter changes